snippets/xen_dom0/snippet.yml

```diff
@@ -7,9 +7,9 @@ boards:
   qemu_cortex_a53:
     append:
       EXTRA_DTC_OVERLAY_FILE: boards/qemu_cortex_a53.overlay
-  rcar_h3ulcb_ca57:
+  rcar_h3ulcb/r8a77951/a57:
```

Comment:
Snippet support is current not working in hwmv2

Reply:
I think it worth to at least mention in commit message or generate error if snippet is used for HWMv2 board, I spent some time yesterday to understand why it is not building properly. In current PR version it will impact 2 boards (salvator_xs and h3ulcb).

Reply:
It is tracked in the main hwmv2 issue here: https://github.com/zephyrproject-rtos/zephyr/issues/51831